squad/core/models.py

```diff
@@ -839,6 +839,8 @@ class Test(models.Model):
 
     __test__ = False
 
+    build = models.ForeignKey(Build, related_name='tests', on_delete=models.CASCADE, null=True)
```

Comment:
are we planning to change the null to false later?

Reply:
yes! It should be mandatory once we got all filled up